testsuite/tests/formatting/margins.ocaml.reference

```diff
@@ -2,12 +2,12 @@ Line 2, characters 4-9:
 2 | 1 + "foo";;
         ^^^^^
 Error: This expression has type
-         string
+         [string]
        but an expression was expected of type
-         int
+         [int]
```

Comment:
Apologies for being the contrarian one, but now I notice that things that are displayed as "code blocks" also get quoted, and I find it a bit frustrating -- I am worried that it could surprise people or lead to misinterpretations.